src/containers/pages/IRS/JurisdictionsReport/index.tsx

```diff
@@ -183,8 +183,8 @@ const JurisdictionReport = (props: GenericJurisdictionProps & RouteComponentProp
   }
 
   const tableProps = {
-    ...(columnsToUse && { columns: columnsToUse }),
     CellComponent: IRSTableCell,
+    columns: [...(columnsToUse || [])],
```

Comment:
Can we have a test where this happens and we check that the resulting columns have actually been generated as expected by the drill down component?  I am not sure that that will happen, but it might

Reply:
Am trying to get a test to crash with the old code now, so we can test this solution

Reply:
Seems like we would need to move this to the helpers file so we can test the actual props object instead of mounting everything and attempting to get the props off the mocked component.